ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -373,8 +373,27 @@ objects:
             exit 1
         fi
 
-        /bin/openshift-install --dir=/tmp/artifacts/installer create cluster &
-        wait "$!"
+        if ! /bin/openshift-install --dir=/tmp/artifacts/installer create cluster
```

Comment:
What about keeping this as one of the steps in `teardown` where we gather artifacts... ?
https://github.com/openshift/release/pull/2581/files#diff-6a0349f9e2bc8f6920d3661f2b81a8e4L388

Reply:
> What about keeping this as one of the steps in teardown where we gather artifacts...

And since we're hoping to have the installer gather these automatically at some point, we probably don't need to bother with "did we succeed?" checks there.  We can just loop through the desired configs and attempt to pull them.  If the bootstrap node is already gone, those attempts will produce empty files, but that's not a big problem.